tests/unit/k8s/K8sInterface_test.cpp

```diff
@@ -42,7 +47,9 @@ TEST_F(K8sInterfaceTest, TestGetServiceList) {
 }
 
 TEST_F(K8sInterfaceTest, TestCreateJasmineGraphWorkerDeployment) {
-    v1_deployment_t *deployment = interface->createJasmineGraphWorkerDeployment(1, "10.43.0.1");
+    v1_node_list_t *nodeList = interface->getNodes();
```

Comment:
Can we have tests for `K8SWorkerController` as well?

Reply:
tests for `K8SWorkerController` are  already there. [https://github.com/miyurud/jasminegraph/blob/master/tests/unit/k8s/K8sWorkerController_test.cpp].
This PR doesn't contain changes that can be tested in `K8SWorkerController`

Reply:
Ok